sequelize/index.d.ts

```diff
@@ -3319,38 +3319,25 @@ declare namespace sequelize {
     /**
      * Options for Model.findOrInitialize method
      */
-    interface FindOrInitializeOptions<TAttributes> extends LoggingOptions {
-
-        /**
-         * A hash of search attributes.
-         */
-        where: string | WhereOptions;
+    interface FindOrInitializeOptions<TAttributes> extends FindOptions {
 
         /**
          * Default values to use if building a new instance
          */
         defaults?: TAttributes;
 
-        /**
-         * Transaction to run query under
-         */
-        transaction?: Transaction;
```

Comment:
@zhengbli because it's already part of `FindOptions` and it has no purpose to do `transaction?: Transaction;` if the same is already on the extended class.